CHANGELOG.md

```diff
@@ -0,0 +1,29 @@
+# Change log
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this
+project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html). See [MAINTAINERS.md](./MAINTAINERS.md)
+for instructions to keep up to date.
+
+
+## v1.0.0
+
+
+#### Project Rename
+
+* The binary name has changed from `sfarweave` to `firearweave` (aligned with https://firehose.streamingfast.io/references/naming-conventions)
+
+#### Flags and environment variables
+
+* Renamed the `mindreader` application to `reader`
+* Renamed `common-blocks-store-url` to `common-block-store-url`
+* Renamed `deepmind` to `firehose`
+* Renamed `debug-deep-mind` flags to `debug-firehose-logs`
+* Renamed `extractor` to `reader`
+* Renamed `DMLOG` prefix to `FIRE`
```

Comment:
No necessity to mention, its internal details, we document stuff that changed that was "public".